yarn.lock

```diff
@@ -1813,6 +1813,17 @@
     lodash-es "4.17.15"
     resize-observer-polyfill "1.5.0"
 
+"@carbon/charts@^0.30.18":
+  version "0.30.24"
+  resolved "https://registry.yarnpkg.com/@carbon/charts/-/charts-0.30.24.tgz#655875b3e2550ba1e0b229ee330b7502ecbd5d49"
+  integrity sha512-VqWPID2iYrJcS0Kf9o7cGLBZPt/Zc3NKSjLzCJ+HixYDQtfljg4YDjdDJy95UrjaKZCNJA7yeEG3w2oqK9iQ7Q==
+  dependencies:
+    "@carbon/colors" "10.4.0"
+    "@carbon/utils-position" "1.1.1"
+    date-fns "2.8.1"
+    lodash-es "4.17.15"
+    resize-observer-polyfill "1.5.0"
+
```

Comment:
I don't think this should be here. I think if you run `yarn` again with the new update to `package.json` it'll remove it?

Reply:
recap our convo - looks like this is due to charts-react having @carbon/charts as a hard dep.

If we pin these deps, we should remove @carbon/charts or add a `resolutions` field to our package.json.

I think we can leave them as caret ranges though for now

Reply:
Reverted to the old ^ version so this diff is gone now 👍